ColorTests.cpp

```diff
@@ -0,0 +1,35 @@
+#include "ColorPair.h"
+
+void testNumberToPair(int pairNumber,
+    TelCoColorCoder::MajorColor expectedMajor,
+    TelCoColorCoder::MinorColor expectedMinor)
+{
+    TelCoColorCoder::ColorPair colorPair = TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+    std::cout << "Got pair " << colorPair.ToString() << std::endl;
+    assert(colorPair.getMajor() == expectedMajor);
+    assert(colorPair.getMinor() == expectedMinor);
+}
+
+void testPairToNumber(
+    TelCoColorCoder::MajorColor major,
+    TelCoColorCoder::MinorColor minor,
+    int expectedPairNumber)
+{
+    int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
+    std::cout << "Got pair number " << pairNumber << std::endl;
+    assert(pairNumber == expectedPairNumber);
+}
+
+void testPrintColorManual()
```

Comment:
This is not a test, but the actual printing of the manual itself.
If someone wanted to print the manual, they would call this function. The "test code" would prove that the function works as expected - prints 25 rows, no duplicate rows, colors in sequence

Reply:
Thank you for the feedback.
Will rename the function & its placement (in a separate file, with meeting LOC limit)
Also, will create the "test code" as pointed out.